### ISSUE TITLE
Create an order

### ISSUE THREAD

**Issue by michaelgold** [orders-squad/orders#5]
**As a** customer
**I need** to be able to create an order
**So that** I can check out the products in my shopping cart

**Assumptions:** 
* The customer needs to choose the payment method
* The customer needs to fill out the shipping address
* There will be an order summary for the customer to review
* There is a place order button 

**Acceptance Criteria:**
```
Given the create an order 
When the customer checks out
Then there will be an order which can be placed
```

**Comment by michaelgold** [comment]
I suggest rewording this user story:

As a customer
I need to be able to create an order
So that I can purchase the items in my shopping cart

**Comment by michaelgold** [comment]
I think the effort level here is a 5

**Comment by EagleTongX** [comment]
I think the effort level here is 8

**Comment by 75divisha** [comment]
I think the effort level here is 8

**Comment by LihengGong** [comment]
Part of the CRUD. I agree with 5.

**Comment by LihengGong** [comment]
Hello guys,
With the latest pull requests merged into master branch, we've already implemented the `Create an Order` function!
To test this function, please first install` HTTPie(brew install httpie)`, then execute the following command:

```shell
http POST :5000/api/orders/ qty=7 price=10.1
```
If everything works fine, the following output is expected:
```shell
[
    {
        "price": "10.1",
        "prod_id": 42,
        "qty": 7
    }
]
```

Please submit a bug if you run into any issues.
